src/index.html

```diff
@@ -2,11 +2,392 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="#">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" rel="stylesheet">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="../src/styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header-logo">
+        <img src="images/logo.png" alt="moyo-logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__ul">
+          <li class="nav__items">
+            <a class="nav__item is-active" href="#">Apple</a>
+          </li>
+
+          <li class="nav__items">
+            <a class="nav__item" href="#">Samsung</a>
+          </li>
+
+          <li class="nav__items">
+            <a class="nav__item" href="#">Smartphones</a>
+          </li>
+
+          <li class="nav__items">
+            <a class="nav__item" href="#"
+               data-qa="nav-hover">Laptops & Computers</a>
+          </li>
+
+          <li class="nav__items">
+            <a class="nav__item" href="#">Gadgets</a>
+          </li>
+
+          <li class="nav__items">
+            <a class="nav__item" href="#">Tablets</a>
+          </li>
+          <li class="nav__items">
+            <a class="nav__item" href="#">Photo</a>
+          </li>
+          <li class="nav__items">
+            <a class="nav__item" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <article class="card" data-qa="card">
+        <img src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        class="card__items card__items--img"
```

Comment:
why you put `card__items` class to all tags? Remove it
```suggestion
        class="card__items--img"
```